types/src/v17/wallet/into.rs

```diff
@@ -655,6 +655,17 @@ impl ListUnspentItem {
     }
 }
 
+impl ListUnspent {
```

Comment:
I'm super anal about code layout in the file. I try to always put things _before_ where they are used, so this should be above the `ListUnspentItem` struct.